src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public abstract class Figure implements AreaCalculation {
+    private String color;
+    private double area;
```

Comment:
We don't need an `area` field for our figures, since if we change a side for example - in our case the area would stay the same, so we have to just calculate it based on the fields that we have.